package.json

```diff
@@ -49,7 +49,7 @@
     "3id-resolver": "^0.0.5",
     "@babel/runtime": "^7.4.5",
     "@ethersproject/hdnode": "5.0.0-beta.137",
-    "@ethersproject/wallet": "5.0.0-beta.136",
+    "@ethersproject/wallet": "^5.0.0-beta.138",
```

Comment:
We should update to an explicitly version (no `^`). Do we want 138 or 139 as in #750?